src/service/routing/directions.clj

```diff
@@ -1,10 +1,31 @@
 (ns service.routing.directions
-  (:require [service.routing.graph.core :as route]
-            [service.routing.osm :as osm]
+  (:require [service.routing.osm :as osm]
             [service.routing.graph.algorithms :as alg]
             [service.routing.graph.protocols :as rp]
-            [service.routing.utils.math :as math]
-            [service.routing.spec :as specs]))
+            [service.routing.libs.math :as math]))
+            ;[service.routing.graph.generators :as g]))
+            ;[clojure.test.check.generators :as gen]))
+            ;[cheshire.core :as cheshire]))
+
+;; https://www.mapbox.com/api-documentation/#stepmaneuver-object
+(def bearing-turns
```

Comment:
@carocad are these values fixed or shall we calculate them in future?

Reply:
no, I just estimated them. Pure empirism. If you find a way to calculate them or even a reference with a standard for those. I am more than happy to change them